### PR TITLE
Scan logs for secrets service

### PR DESCRIPTION
**Pull request by dekiel**
**Description**

Changes proposed in this pull request:

Cloud run service scanning prowjob logs stored in public available gcp bucket.
Service is triggered by pubsub message send by prow.
Service accepts cloudevents format events and reply in the same format.
Scanning is done using gitleaks library. Library allow for adding custom scanning rules.

### CODE REVIEW

File: development/gcp/cloud-run/scan-logs-for-secrets/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.19.3-alpine3.16 as builder
```
Comment (halamix2) [inline development/gcp/cloud-run/scan-logs-for-secrets/Dockerfile:1]: Could you bump this image or add this to the dependabot?

**Review by Sawthis — APPROVED** [review]

File: development/gcp/cloud-run/scan-logs-for-secrets/Dockerfile (hunk 1 above)
Comment (dekiel) [reply development/gcp/cloud-run/scan-logs-for-secrets/Dockerfile]: Updated to 1.19.5-alpine3.17

**Pull request merged**